akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -439,6 +515,12 @@ final class ClusterClientReceptionist(system: ExtendedActorSystem) extends Exten
     pubSubMediator ! DistributedPubSubMediator.Unsubscribe(topic, actor)
 
   /**
+   * Returns the underlying receptionist actor.
+   */
+  def underlying: ActorRef =
+    receptionist
```

Comment:
Useful to determine that events being emitted by a receptionist are the ones associated with the receptionist that you want (you can technically have many receptionists).